File: NetBeansProjects/Quiz/src/main/java/quiz.java
import java.util.*;
public class quiz {
    public static void main(String[] args){
       Scanner scan = new Scanner(System.in);
       String ans = scan.nextLine();
       String ans2 = scan.nextLine();
       question1(ans);
       question2(ans2);
       int score = question1(ans) + question2(ans2);
       System.out.println("Your Score Is :- " + score);
    }
    public static int question1(String a){
        System.out.println("Q1) Which Of These Is Not A Browser ?");
        System.out.println("A) Facebook");
        System.out.println("B) Chrome");
        System.out.println("C) Firefox");
        System.out.println("D) Opera");
        //Scanner scan = new Scanner(System.in);
        //String a = scan.nextLine();
        int score = 0;
        switch(a){
            case "A":
                score += 10;
                break;
            case "B":break;
            case "C":break;
            case "D":break;
            default:
                System.out.println("You Have Done Something Wrong");
                break;
        }
        return score;
    }
    public static int question2(String a){
        System.out.println("Q1) Who Is The CEO Of Google ?");
        System.out.println("A) Mark Zuckerburg");
        System.out.println("B) Sunder Pichai");
        System.out.println("C) Tim Cook");
        System.out.println("D) Bill Gates");
        //Scanner scan = new Scanner(System.in);
        //String a = scan.nextLine();
        int score = 0;
        switch(a){
            case "A":break;
            case "B":
                score += 10;
                break;
            case "C":break;
            case "D":break;
            default:
                System.out.println("You Have Done Something Wrong");
                break;
        }
        return score;
    }
    public static int question3(){
        System.out.println("Q1) Which Of These is The New Operating System Getting Devloped By Google?");
        System.out.println("A) Fuchisia");
        System.out.println("B) Flutter");
        System.out.println("C) Sailfish");
        System.out.println("D) Tizen");
        Scanner scan = new Scanner(System.in);
        String a = scan.nextLine();
        int score = 0;
        switch(a){
            case "A":
                score += 10;
                break;
            case "B":break;
            case "C":break;
            case "D":break;
            default:
                System.out.println("You Have Done Something Wrong");
                break;
        }
        return score;
    }
}

File: PycharmProjects/Quiz/Quiz.py
def question1():
    score1 = 0
    print("Q1) Which Of These Is Not A Browser ?")
    print("A) Facebook")
    print("B) Chrome")
    print("C) Firefox")
    print("D) Opera")
    ans = input("What Is the Answer ?")
    if ans == "A" or ans == 'a':
        score1 += 10
    elif ans == "B" or ans == 'b':
        score1 += 0
    elif ans == "C" or ans == 'c':
        score1 += 0
    elif ans == "D" or ans == 'd':
        score1 += 0
    else:
        print("You Did Some Thing Wrong")
    return score1


def question2():
    score2 = 0
    print("Q2) Who Is The CEO Of Google ?")
    print("A) Mark Zuckerburg")
    print("B) Sunder Pichai")
    print("C) Tim Cook")
    print("D) Bill Gates")
    ans = input("What Is the Answer?")
    if ans == "A" or ans == 'a':
        score2 += 0
    elif ans == "B" or ans == 'b':
        score2 += 10
    elif ans == "C" or ans == 'c':
        score2 += 0
    elif ans == "D" or ans == 'd':
        score2 += 0
    else:
        print("You Did Some Thing Wrong")
    return score2


ch = 'Y'
while ch == 'Y' or ch == 'y':
    score = question1()
    print()  # This IS For Next Line
    score += question2()
    print()  # This IS For Next Line
    print("Your Score is : ", score)
    print()  # This IS For Next Line
    if score == 20:
        print("Very Good Work You Answered All Answer Correctly!!!")
    elif score == 10:
        print("Good But You Need More Knowledge!!!")
    else:
        print("You Need To Work On Your G.K.")
    print()  # This IS For Next Line
    ch = input("Do You Want To Retry ?(Y for Yes And N for No)")
    print()  # This IS For Next Line
